services/ui-src/src/components/cards/EntityCard/EntityCardTopSection.tsx

```diff
@@ -99,7 +99,11 @@ export const EntityCardTopSection = ({
                   ? "D2.VII.7a Reporting Period and D2.VII.7b Reporting period: Date range"
                   : "Measure Reporting Period"}
               </Text>
-              <Text sx={sx.subtext}>{formattedEntityData.reportingPeriod}</Text>
+              <Text sx={sx.subtext}>
+                {formattedEntityData.reportingPeriod === "Yes"
+                  ? formattedEntityData.reportingPeriod
+                  : `No, ${formattedEntityData.reportingPeriod}`}
```

Comment:
Note: I would change this where it's being formatted initially here instead: https://github.com/CMSgov/mdct-mcr/blob/0bc7d19449faa3737ff21e3341a5c73e58f6cdcf/services/ui-src/src/utils/reports/entities.ts#L26-L30